src/main/java/com/powsybl/openloadflow/network/VoltageControl.java

```diff
@@ -73,11 +73,15 @@ public Type getType() {
         return type;
     }
 
+    /**
+     * Check is the merged voltage to which this voltage control belongs is disabled. Disabled means that there is no
+     * more controlled bus or no more controller element.
```

Comment:
We must add in the comment that loosing a controlled bus in a merge voltage control is an open question. We can also imagine that loosing a controlled bus leads to the loose of its controllers and maybe a different management of the merge status.